mix.exs

```diff
@@ -101,10 +105,17 @@ defmodule MongoosePush.Mixfile do
 
   # All mix tasks are redundant in runtime, but we still need to compile `lib/mix/tasks/compile_asn1.ex`
   # as it's required by build process (ASN1 compiler).
-  defp elixirc_paths(:prod),
-    do: ["lib"]
+
+  defp elixirc_paths(:prod), do: ["lib"]
 
   defp elixirc_paths(:test), do: ["lib", "test/support"]
 
+  defp elixirc_paths(:integration), do: ["test/support"]
+
   defp elixirc_paths(_), do: ["lib"]
+
+  # Add "test/integration" path after the https://erlangsolutions.atlassian.net/browse/MIM-748 is resolved
```

Comment:
Let's configure this file properly and use `@skip` tag instead to skip not-working tests.